enterprise/api_client/enterprise_catalog.py

```diff
@@ -200,14 +200,35 @@ def delete_enterprise_catalog(self, catalog_uuid):
             )
             return {}
 
+    def traverse_get_content_metadata(self, endpoint, query, catalog_uuid):
+        """
+        Helper method to traverse over a paginated response from the enterprise-catalog service's `get_content_metadata`
+        endpoint.
+        """
+        content_metadata = OrderedDict()
+        try:
+            response = endpoint.get(**query)
+            for item in utils.traverse_pagination(response, endpoint):
```

Comment:
oh wait where do you even use the catalog_uuid?

Reply:
`endpoint = getattr(self.client, self.GET_CONTENT_METADATA_ENDPOINT.format(catalog_uuid))` catalog uuid is part of the URL, which is included in the endpoint.